enterprise/backend/src/metabase_enterprise/audit_app/pages/common.clj

```diff
@@ -285,8 +285,8 @@
 
 (defn- format-separator
   [_separator [x y]]
-  (let [[x-sql & x-args] (sql/format-expr x)
-        [y-sql & y-args] (sql/format-expr y)]
+  (let [[x-sql & x-args] (sql/format-expr x {:nested true})
```

Comment:
`{:nested true}` makes sure the expression is generated in parens when applicable, e.g.

```clj
(honey.sql/format-expr [:+ 1 2])
=> ["1 + 2"]

(honey.sql/format-expr [:+ 1 2] {:nested true})
=> ["(1 + 2)"]
```

I don't think we actually need to be doing this here because whatever `x` and `y` are happen to work without it, but this is more correct. I went thru all the places we were using `format-expr` and added it to be safe